symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/DatafeedExampleMain.java

```diff
@@ -18,14 +17,13 @@ public static void main(String[] args) throws Exception {
     final SymphonyBdk bdk = new SymphonyBdk(loadFromSymphonyDir("config.yaml"));
 
     bdk.datafeed().subscribe(new RealTimeEventListener() {
+
       @Override
       public void onMessageSent(V4Initiator initiator, V4MessageSent event) {
         bdk.messages().send(event.getMessage().getStream(), "<messageML>Hello, World!</messageML>");
       }
     });
 
-    bdk.activities().register(slash("/hello", context -> bdk.messages().send(context.getStreamId(), "<messageML>Hello, World!</messageML>"), "Hello Command"));
```

Comment:
Why was it removed ? Was the slash command example irrelevant ?

Reply:
Yes, we already provide a slash command example in https://github.com/finos/symphony-bdk-java/blob/main/symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/activity/GifActivityMain.java#L49